addon/components/bs4/bs-form/element/label.js

```diff
@@ -1,9 +1,65 @@
+import Ember from 'ember';
 import FormElementLabel from 'ember-bootstrap/components/base/bs-form/element/label';
 
+const {
+  computed
+} = Ember;
+
 export default FormElementLabel.extend({
   tagName: 'label',
 
-  classNames: ['form-control-label'],
-  classNameBindings: ['invisibleLabel:sr-only', 'labelClass'],
-  attributeBindings: ['formElementId:for']
+  classNames: [],
+  classNameBindings: ['invisibleLabel:sr-only', 'isHorizontal:col-form-label', 'isCheckbox:form-check-label', 'labelClass'],
```

Comment:
The change to `col-form-label` seems to overlap https://github.com/kaliber5/ember-bootstrap/pull/260, and is subject to https://github.com/twbs/bootstrap/issues/22173 getting clarified. But this might be ok for now?